src/program_header/mod.rs

```diff
@@ -14,6 +14,7 @@ bitflags! {
     /// for 64-bit ELFs.
     ///
     /// Also called "Segment Permissions" in ELF specification or "p_flags".
+    #[derive(Debug)]
```

Comment:
nit: this could also be Copy, Clone, PartialEq and Eq

Reply:
Good point, I derived the common traits as well.